test/e2e/kubernetes/workloads/sgx-device-plugin.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: sgx-device-plugin
+  namespace: kube-system
```

Comment:
The namespace in the spec doesn't match the namespace we're looking for in the test ("default"). In general, we should put these out of kube-system namespace, unless there's a reason to. (In which case we'd change the test to validate the pod creation in the "kube-system" namespace).

Reply:
I feel stupid, thanks for pointing that out

Reply:
I recommend feeling silly or something positive instead!